include/executor/engine/memory.ipp

```diff
@@ -131,7 +131,11 @@ Executor::runLoadSplatOp(Runtime::StackManager &StackMgr,
   uint32_t EA = Val.get<uint32_t>() + Instr.getMemoryOffset();
 
   // Value = Mem.Data[EA : N / 8]
+#if defined(_MSC_VER) && !defined(__clang__) // MSVC
```

Comment:
This could be replaced with the above modifications.

Reply:
Fixed in https://github.com/WasmEdge/WasmEdge/pull/2751/commits/83d3d8e1c4cc1e2f5c67eb3d4aad6ba9cd778fc4#diff-7f1822f82a75e31952730d9cbe7d4891290088cd65ca9b19c5a8b5c03068a80c